geometry/matrix_3.c

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file
+ * @brief 3x3 Matrix operations
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 😄